tests/process/spawn.luau

```diff
@@ -141,8 +141,10 @@ assert(
 
 -- Passing stdin strings should work
 
-local stdinChild = process.spawn("xargs", {
-	"echo",
+local isWindows = process.os == "windows"
```

Comment:
This could lead to a duplicate variable.

Reply:
Yeah, I hadn't synced with the main branch at that time, will fix.

Reply:
Fixed in https://github.com/filiptibell/lune/pull/106/commits/9bc39d3aef14a503bd0af0a0084318d841f6d01e.